### PR TITLE
Fix windows support

### PR DESCRIPTION
**Pull request by tsing80**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = {
   name: 'Ember CLI ic-ajax',
 
   init: function(name) {
-    this.treePaths['vendor'] = require.resolve('ic-ajax').replace('ic-ajax/dist/cjs/main.js', '');
+    this.treePaths['vendor'] = require.resolve('ic-ajax').replace(require('path').join('ic-ajax','dist','cjs','main.js'), '');
```
Comment (rwjblue) [inline index.js]: Can you split the `path.join` bit into a separate local variable above this?

**Comment by tsing80** [comment]
Reworked, it still does not work though for me both in windows and mac in npm 3 before and after the change

**Comment by rwjblue** [comment]
@tsing80 - NPM 3 requires changes in ember-cli also (submitted in https://github.com/ember-cli/ember-cli/pull/4349).

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**